share/instruments/instruments.xml

```diff
@@ -7410,7 +7409,7 @@
                         <head>cross</head>
                         <line>2</line>
                         <voice>0</voice>
-                        <name>Mute High Conga </name>
+                        <name>Mute High Conga</name>
```

Comment:
Clash with #7893, which changes this string to "Mute Hi Conga" (and with the trailing space, I'm going to fix that though)

Reply:
Guess we'd need to either adjust the names in the spreadsheet or the strings in importgtp.cpp (and maybe drumset.cpp).
Or find another way to get these translated in the Edit Drumset dialog.

Reply:
I have adopted your changes from #7893 except for "Hi", which I have kept as "High" for now because it seems to be the more common variant. I have kept "Hi" where it is followed by a hyphen (e.g. "Hi-hat").

Reply:
Then we'd have to make sure to update them in 
https://github.com/musescore/MuseScore/blob/6fe71f9ed129e502b0ab4030d4672833efbbd6de/src/libmscore/drumset.cpp#L256-L312 and the more complete https://github.com/musescore/MuseScore/blob/6fe71f9ed129e502b0ab4030d4672833efbbd6de/src/importexport/guitarpro/internal/importgtp.cpp#L303-L373